integration/token/fungible/views/swap.go

```diff
@@ -86,11 +86,13 @@ func (t *SwapInitiatorView) Call(context view.Context) (interface{}, error) {
 
 	outputs, err := tx.Outputs()
 	assert.NoError(err, "failed getting outputs")
-	os := outputs.ByRecipient(other)
+	// get outputs from Bob filtered by Alice type
```

Comment:
... by the type of tokens received from Alice.

Reply:
okay